university-of-roehampton-harvard.csl

```diff
@@ -429,7 +429,7 @@
       </if>
       <else>
         <names variable="author">
-          <name form="short" and="text" delimiter-precedes-last="never" initialize-with="."/>
+          <name form="short" and="text" delimiter-precedes-last="never" et-al-min="4" et-al-use-first="1" initialize-with="."/>
```

Comment:
Can you please move the et al into the <citation... line, where it applies to all citations (and is easier to find)